docs/arch/microtvm_project_api.rst

```diff
@@ -0,0 +1,151 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _microtvm_project_api:
+
+microTVM Project API
+====================
+
+About microTVM Project API
+--------------------------
+
+The microTVM Project API allows TVM to automatically run models on
+unconventional platforms. It allows platforms to define a standard function to
```

Comment:
I think the use of `unconventional` here makes this sentence a bit vague, perhaps we could just say _embedded_ (?), given the paragraph goes towards producing a _firmware_, which then makes everything more specific.

Reply:
added "unconventional or embedded," maybe that helps. agree unconventional is a bit vague but given the example make flow i do think it's a bit limiting. the concept i want to convey here is "deploy in scenarios where you have to build code using an external build tool."

Reply:
ack. It is more clear now.